test/args-test.js

```diff
@@ -0,0 +1,30 @@
+import { test } from 'ava'
+
+import Args from '../lib/args'
+
+test('encode empty', t => {
+  const args = {}
+  const encoded = Args.encode(args)
+  t.deepEqual(encoded, '%7B%7D')
+})
+
+test('file with query string', t => {
+  const url = '/test/file.html?p=hello&n=world'
+  const args = {}
+  const newUrl = Args.urlWithArgs(url, args)
+  t.deepEqual(newUrl, 'file:///test/file.html?p=hello&n=world#%7B%7D')
```

Comment:
it doesn't look like args is required, and if you don't pass an object it should use an empty string.
so can we update this test to:
```
const newUrl = Args.urlWithArgs(url)
t.deepEqual(newUrl, 'file:///test/file.html?p=hello&n=world')
```
Isn't this your original use case? Thanks! this looks great otherwise.